examples/experimental/dagster-airlift/dagster_airlift_tests/unit_tests/core_tests/test_sensor.py

```diff
@@ -30,7 +31,7 @@ def make_dag_key_str(dag_id: str) -> str:
     return make_dag_key(dag_id).to_user_string()
 
 
-def test_dag_and_task_metadata(init_load_context: None) -> None:
+def test_dag_and_task_metadata(init_load_context: None, instance: DagsterInstance) -> None:
```

Comment:
where is this fixture even coming from

Reply:
examples/experimental/dagster-airlift/dagster_airlift_tests/unit_tests/conftest.py

pytest is crazy